README.md

```diff
@@ -29,9 +29,95 @@ For existing repositories in the Keptn Sandbox Organization the respective code
 
 ## How to be promoted to Keptn Contrib organization?
 
-The [Keptn Contrib](https://github.cmo/keptn-contrib) holds projects that have been promoted from Keptn Sandbox due to their adoption and high quality standards.
+The [Keptn Contrib](https://github.com/keptn-contrib) holds projects that have been promoted from Keptn Sandbox due to their adoption and high quality standards.
 To suggest your contribution to the Keptn Contrib organization, it has to fulfull all criteria listed above, plus the following:
 1. Automated builds
 1. Automated tests that are validated for each build
 1. At least one release of the service compatible with the latest Keptn version
 1. At least one sponsor of the Keptn core team for your contribution
+
+## RBAC Guidelines
+* Use the Service Account `keptn-default` of the keptn installation namespace wherever possible. This account has no k8s permissions assigned.
+* Never apply additional Roles or Rolebindings on the Service Account `keptn-default`
+* If your service needs permissions on k8s, create a new service account with the name `keptn-<your-service-name>` in the keptn installation namespace
+* Apply Roles, ClusterRoles, Rolebindings and ClusterRoleBindings with a minimum set of useful permissions on that service account
+* Prefer namespaced Roles and Rolebindings over ClusterRoles and ClusterRoleBindings
+
+### Example:
+Let's image, the service (deployment) `keptn-sample-service` needs read access for the secret "keptn-sample-secret".
```

Comment:
we need to clarify this sentence, e.g.,:
Let's imagine the Keptn service (which corresponds to a Kubernetes deployment and service) named `keptn-sample-service` requires read-access for a secret called `keptn-sample-secret` in the `keptn` namespace.

Reply:
Thanks, addressed.